include/openssl/evp.h

```diff
@@ -355,6 +355,10 @@ int (*EVP_CIPHER_meth_get_ctrl(const EVP_CIPHER *cipher))(EVP_CIPHER_CTX *,
 # define         EVP_CTRL_GET_IV                         0x25
 /* Tell the cipher it's doing a speed test (SIV disallows multiple ops) */
 # define         EVP_CTRL_SET_SPEED                      0x26
+/* Set the IV used by the cipher */
+# define         EVP_CTRL_SET_IV                         0x27
+/* Get the tag length for AEAD cipher */
+# define         EVP_CTRL_AEAD_GET_TAGLEN                0x28
```

Comment:
These ctrls need to be documented

Reply:
Will add it. I'm still not sure if we need `EVP_CTRL_SET_IV` as mentioned above. Will need to check it first.

Reply:
So, I guess this is still outstanding.

Reply:
This has been removed as it's no longer needed.